src/ppx/application.cpp

```diff
@@ -1305,8 +1340,9 @@ int Application::Run(int argc, char** argv)
             mAverageFrameTime = static_cast<float>(nowMs / mFrameCount);
         }
 
-        // Update the metrics.
-        UpdateMetrics();
+        // Update the metrics. This can be used for both recorded AND displayed metrics,
+        // and therefore should always be called.
+        DispatchUpdateMetrics();
```

Comment:
nit: why `dispatch`? I see this a LOT around BigWheels, but we just `UpdateSomething()`. Isn't dispatch used when there is some kind of scheduling involved? Or is there some English subtillity I miss?

Reply:
TBH I'm just copying the existing style; I agree, I think dispatch is pretty meaningless in a lot of what we're doing. I also don't know why these functions are all virtuals, when it appears they're used as the internal wrapper methods to CALL virtuals, and are never overridden by anything.

So basically I suspect this just needs some cleanup, but I wasn't willing to do that here. I figured using 'Dispatch' was enough of a flag to someone that it required the same cleanup consideration.

Reply:
As long as we all know we do it "until we clean this up", I'm fine :)